static/app/components/forms/selectOption.tsx

```diff
@@ -12,47 +13,52 @@ function SelectOption(props: Props) {
   const {label, data, selectProps, isMulti, isSelected, isFocused, isDisabled} = props;
   const {showDividers, verticallyCenterCheckWrap} = selectProps;
   const {
+    value,
     details,
+    tooltip,
+    tooltipOptions = {delay: 500},
     leadingItems,
     trailingItems,
     leadingItemsSpanFullHeight,
     trailingItemsSpanFullHeight,
   } = data;
 
   return (
-    <selectComponents.Option {...props} className="select-option">
-      <InnerWrap isFocused={isFocused} isDisabled={isDisabled}>
-        <Indent isMulti={isMulti} centerCheckWrap={verticallyCenterCheckWrap}>
-          <CheckWrap isMulti={isMulti} isSelected={isSelected}>
-            {isSelected && (
-              <IconCheckmark
-                size={isMulti ? 'xs' : 'sm'}
-                color={isMulti ? 'white' : undefined}
-              />
+    <selectComponents.Option className="select-option" {...props}>
+      <Tooltip containerDisplayMode="block" title={tooltip} {...tooltipOptions}>
```

Comment:
Can you use the `skipWrapper` prop here?

Reply:
Oh nice didn't know that was available! - https://github.com/getsentry/sentry/pull/33983/commits/4d1447f6ac5107ed10dbb6f59f81fbc7d281ff9a